P3.txt

```diff
@@ -0,0 +1,16 @@
+P3. Compute the sum of n numbers
+
+val n: Int
+read(n)
+
+if (n <= 0)
+	write(“Can’t be done”)
+else {
+	val arr: Int[n]
+	for (0 to n - 1)
+		read(arr[i])

```

Comment:
Firstly, the array elements' access is not defined (e.g., arr[i]). By the syntax definition, ioStatement = "read" "(" identifier ")" , so the read opration cannot have as parameter an array access element operation.